### PR TITLE
Upgrade some APIs to 2.0.

### PR DESCRIPTION
**Pull request by hutuxian**
Upgrade some APIs to 2.0.

### CODE REVIEW

File: docs/source/paddle_fleet_rst/fleet_static_quick_start_cn.rst [hunk 1]
```diff
@@ -23,19 +23,21 @@ Training），接下来的例子会以同样的模型来说明两种架构的数
...
-       y = paddle.data(name="y", shape=[64, 1], dtype='int64')
-       x_flatten = paddle.reshape(x, [64, 784])
-       fc_1 = nn.fc(input=x_flatten, size=128, act='tanh')
-       fc_2 = nn.fc(input=fc_1, size=128, act='tanh')
-       prediction = nn.fc(input=[fc_2], size=10, act='softmax')
-       cost = paddle.fluid.layers.cross_entropy(input=prediction, label=y)
-       acc_top1 = paddle.fluid.layers.accuracy(input=prediction, label=y, k=1)
-       avg_cost = paddle.fluid.layers.mean(x=cost)
-       return train_dataset, test_dataset, x, y, avg_cost, acc_top1
+      # paddle.vision.set_image_backend('cv2')
```
Comment (danleifeng) [inline docs/source/paddle_fleet_rst/fleet_static_quick_start_cn.rst]: 注释去掉？

File: docs/source/paddle_fleet_rst/fleetrun_usage_cn.rst [hunk 2]
```diff
@@ -240,60 +240,61 @@ fleetrun命令参数介绍
...
-                       % (i, step, loss_val[0], total_time,
-                          1 / (end_time - start_time) ))
+      import os
+      import time
+      import paddle
+      import paddle.distributed.fleet as fleet
+      import paddle.static.nn as nn
+      import paddle.fluid as fluid
+
+      paddle.vision.set_image_backend('cv2')
```
Comment (danleifeng) [inline docs/source/paddle_fleet_rst/fleetrun_usage_cn.rst]: set_image_backend需要放在最前面吗？移到enable_static之后？

File: docs/source/paddle_fleet_rst/fleetrun_usage_cn.rst [hunk 3]
```diff
@@ -240,60 +240,61 @@ fleetrun命令参数介绍
...
+      def mnist_on_mlp_model():
+          train_dataset = paddle.vision.datasets.MNIST(mode='train')
+          test_dataset = paddle.vision.datasets.MNIST(mode='test')
+          x = paddle.static.data(name="x", shape=[64, 1, 28, 28], dtype='float32')
```
Comment (danleifeng) [inline docs/source/paddle_fleet_rst/fleetrun_usage_cn.rst]: 修改下组网shape

File: docs/source/paddle_fleet_rst/fleet_static_quick_start_cn.rst (hunk 1 above)
Comment (hutuxian) [reply docs/source/paddle_fleet_rst/fleet_static_quick_start_cn.rst]: done

File: docs/source/paddle_fleet_rst/fleetrun_usage_cn.rst (hunk 2 above)
Comment (hutuxian) [reply docs/source/paddle_fleet_rst/fleetrun_usage_cn.rst]: done

File: docs/source/paddle_fleet_rst/fleetrun_usage_cn.rst (hunk 3 above)
Comment (hutuxian) [reply docs/source/paddle_fleet_rst/fleetrun_usage_cn.rst]: done

File: docs/source/paddle_fleet_rst/fleet_static_quick_start_cn.rst [hunk 4]
```diff
@@ -23,19 +23,20 @@ Training），接下来的例子会以同样的模型来说明两种架构的数
...
-       return train_dataset, test_dataset, x, y, avg_cost, acc_top1
+      train_dataset = paddle.vision.datasets.MNIST(mode='train')
+      test_dataset = paddle.vision.datasets.MNIST(mode='test')
+      x = paddle.static.data(name="x", shape=[64, 28, 28], dtype='float32')
+      y = paddle.static.data(name="y", shape=[64, 1], dtype='int64')
+      x_flatten = paddle.reshape(x, [64, 784])
+      fc_1 = nn.fc(x=x_flatten, size=128, activation='tanh')
+      fc_2 = nn.fc(x=fc_1, size=128, activation='tanh')
+      prediction = nn.fc(x=[fc_2], size=10, activation='softmax')
+      cost = paddle.fluid.layers.cross_entropy(input=prediction, label=y)
```
Comment (seiriosPlus) [inline docs/source/paddle_fleet_rst/fleet_static_quick_start_cn.rst:36]: 这里还有 Fluid

File: docs/source/paddle_fleet_rst/fleet_static_quick_start_cn.rst [hunk 5]
```diff
@@ -55,12 +56,12 @@ Training），接下来的例子会以同样的模型来说明两种架构的数
    place = paddle.CUDAPlace(int(os.environ.get('FLAGS_selected_gpus', 0)))
    train_dataloader = paddle.io.DataLoader(
        train_data, feed_list=[x, y], drop_last=True,
-       places=place, batch_size=64, shuffle=True)
-   fleet.init(is_collective=True)
+       places=place, batch_size=64, shuffle=True, return_list=False)
    strategy = fleet.DistributedStrategy()
+   fleet.init(is_collective=True, strategy=strategy)
    #optimizer = paddle.optimizer.Adam(learning_rate=0.01)
    optimizer = paddle.fluid.optimizer.Adam(learning_rate=0.001)
```
Comment (seiriosPlus) [inline docs/source/paddle_fleet_rst/fleet_static_quick_start_cn.rst]: 这里

Comment (hutuxian) [reply docs/source/paddle_fleet_rst/fleet_static_quick_start_cn.rst]: done

File: docs/source/paddle_fleet_rst/fleet_static_quick_start_cn.rst (hunk 4 above)
Comment (hutuxian) [reply docs/source/paddle_fleet_rst/fleet_static_quick_start_cn.rst:36]: 等用法确定再更新

**Review by seiriosPlus — APPROVED** [review]

**Comment by seiriosPlus** [comment]
LGTM

**Pull request merged**